test/models/report_test.rb

```diff
@@ -3,7 +3,60 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @user = users(:alice)
+    @report = reports(:alice_report)
+    @mentioned_report = reports(:bob_report)
+  end
+  test 'editable?は@userが日報作成者ならtrueを返す' do
+    assert @report.editable?(@user)
+  end
+
+  test 'editable?は@userが日報作成者じゃないならfalseを返す' do
+    incorrect_user = users(:bob)
+    assert_not @report.editable?(incorrect_user)
+  end
+
+  test 'created_onはDateクラスを返す' do
+    @report.created_at = Time.zone.now
+    assert_instance_of Date, @report.created_on
+  end
+
+  test '日報の保存時は既存の言及を全削除し、新規の言及を作成する' do
+    assert_equal 1, @report.active_mentions.size
+    assert @report.active_mentions.find_by(mentioned_by_id: @mentioned_report.id)
```

Comment:
```suggestion
    assert_equal [@mentioned_report], @report.mentioning_reports
```

ってするとシンプルで意図が明確になりそうです
（34-37行目も同様）

Reply:
ありがとうございます！
[関連付けがなくなったことを確認するテストがアサーションによって挙動が異なる](https://bootcamp.fjord.jp/questions/1821)と同じ現象が起こったので`reload`を使用しました。